### DISCUSSION TITLE
How to format number in input component

### DISCUSSION THREAD

**Discussion by ghost** [unovue/shadcn-vue#594] [Q&A] [answered]
I am using a form with one input component. Currently, when I enter a number, 
It displays as: 100000000.
Desired format: 100,000,000.

**Comment by sadeghbarati** [comment, marked as answer]
You need to use `format-options` prop in `NumberField` component 

eg. https://www.shadcn-vue.com/docs/components/number-field.html#currency

**Reply by ghost** [reply]
Thank you for the response. Last time, I tried using `NumberField` but encountered an error because **I hadn't restarted Vite**. Now, my issue has been resolved. Thanks again!
